src/components/layout/Flex.js

```diff
@@ -24,11 +25,15 @@ const flexPropTypes = {
   wrap: PropTypes.bool,
 };
 
+const validatorFn = memoFn(prop => {
+  return !Object.keys(flexPropTypes).includes(prop);
```

Comment:
If we're trying to optimise this, it might be worth pulling the `Object.keys` call out into its own variable since `flexPropTypes` is static.